app/models/article.rb

```diff
@@ -88,16 +88,23 @@ class Article < ApplicationRecord
   scope :published, -> { where(published: true) }
   scope :unpublished, -> { where(published: false) }
 
+  scope :latest_published_admin_thread_with, lambda { |tag_name|
+    published.
+      where(user_id: SiteConfig.staff_user_id).
+      order("published_at ASC").
+      tagged_with(tag_name).last
```

Comment:
In this case, the scope will return an `ActiveRecord_Relation` instead of an instance of an `Article`.
Then `daily_thread.path` will fail with a `NoMethodError`.  I would convert a scope into a method to solve this, and some kind of a test would be useful to catch such errors.

Reply:
@lightalloy I think there might be some value in keeping this as a scope. I've updated the method name to be more semantic and applied `.first` to all the appropriate calls. Let me know what you think.